pkg/bot/config.go

```diff
@@ -48,6 +49,18 @@ type Cluster struct {
 	Kubeconfig string `yaml:"kubeconfig"`
 }
 
+type PostProductionStep struct {
+	Command string `yaml:"command"`
+	Cluster string `yaml:"cluster"`
+}
+
+func (c *Component) HasPostProductionStep() bool {
```

Comment:
Just realized that postDeploymentStep might be more suitable.

Reply:
+1